src/graph.rs

```diff
@@ -1735,16 +1817,7 @@ impl<'a, 'graph> Builder<'a, 'graph> {
     loop {
       let specifier = match self.pending.next().await {
         Some((specifier, maybe_referrer, Ok(Some(response)))) => {
-          let assert_types =
-            self.pending_specifiers.remove(&specifier).unwrap();
-          for maybe_assert_type in assert_types {
-            self.visit(
-              &specifier,
-              &response,
-              maybe_assert_type,
-              maybe_referrer.clone(),
-            )
-          }
```

Comment:
This is the current method for handling different assertions being made against the same module... re-parse it with each one (because that's where assertion errors are generated). And it seems like there's no uniqueness in the assert types in `pending_specifiers` because it includes the range... so are we re-parsing the module for every import that was found for it until it was visited?